src/Tribe/Tickets_Handler.php

```diff
@@ -93,9 +98,10 @@ public function __construct() {
 		foreach ( $main->post_types() as $post_type ) {
 			add_action( 'save_post_' . $post_type, array( $this, 'save_image_header' ) );
 			add_action( 'save_post_' . $post_type, array( $this, 'save_global_stock' ) );
-			add_action( 'save_post_' . $post_type, array( $this, 'save_tickets_order' ) );
 		}
 
+		add_action( 'save_post_tribe_events', array( $this, 'save_tickets_order' ) );
```

Comment:
Could we use the `Tribe__Events__Main::POSTTYPE` here instead, and concatenate it to `save_post_`? I hope we never change that constants value, but in case we ever do its nice to use its value rather than just the `tribe_events` string.